src/index.js

```diff
@@ -1,107 +1,39 @@
 
-function display(task, ind) {
-  const ulListTask = document.getElementById('list-task');
-  const liTask = document.createElement('li');
-  liTask.classList.add('task');
-  liTask.id = task[ind - 1].index;
+/* List Task */
+const listTask = new ListTasks();
 
-  const divTaskContent = document.createElement('div');
-  divTaskContent.classList.add('align-center', 'task-content');
-  const checkBok = document.createElement('input');
-  checkBok.type = 'checkbox';
-  checkBok.name = task[ind - 1].index;
-
-  const inTaskDescription = document.createElement('input');
-  inTaskDescription.classList.add('task-description');
-  inTaskDescription.value = task[ind - 1].description;
-  inTaskDescription.type = 'text';
-
-  checkBok.addEventListener('change', (event) => {
-    const { id } = event.currentTarget.parentElement.parentElement;
-    if (event.currentTarget.checked) {
-      event.currentTarget.nextSibling.style.textDecoration = 'line-through';
-      this.list[id - 1].completed = true;
-    } else {
-      event.currentTarget.nextSibling.style.textDecoration = 'none';
-      this.list[id - 1].completed = false;
-    }
-  });
-
-  if (task.completed) {
-    checkBok.click();
-    inTaskDescription.style.textDecoration = 'line-through';
+/* On submit the input */
+const form = document.forms[0];
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  if (form.newtask.value.trim().length !== 0) {
+    listTask.add(form.newtask.value);
   }
+  form.newtask.value = '';
+});
 
-  inTaskDescription.addEventListener('focus', (event) => {
-    const { id } = event.currentTarget.parentElement.parentElement;
-    document.getElementById(id).style.backgroundColor = 'rgb(156, 156, 255)';
-    const iDelete = event.currentTarget.parentElement.nextSibling;
-    const iMenu = event.currentTarget.parentElement.nextSibling.nextSibling;
-    iDelete.classList.remove('hidden');
-    iMenu.classList.add('hidden');
-  });
-
-  inTaskDescription.addEventListener('focusout', (event) => {
-    const { id } = event.currentTarget.parentElement.parentElement;
-    const litask = document.getElementById(id);
-    litask.style.backgroundColor = 'white';
-
-    this.list[id - 1].description = event.currentTarget.value;
-    const iDelete = event.currentTarget.parentElement.nextSibling;
-    const iMenu = event.currentTarget.parentElement.nextSibling.nextSibling;
-
-    setTimeout(() => {
-      iMenu.classList.remove('hidden');
-      iDelete.classList.add('hidden');
-    }, 200);
-  });
+const btnClearAll = document.getElementById('btn-clear-task');
+btnClearAll.addEventListener('click', () => {
+  listTask.clearAll();
+});
 
-  divTaskContent.append(checkBok);
-  divTaskContent.append(inTaskDescription);
+/* Save changes in Local Storage */
+window.addEventListener('beforeunload', () => {
+  const savedData = { currentDescription: form.newtask.value, list: listTask.list };
+  window.localStorage.setItem('SavedData', JSON.stringify(savedData));
+});
 
```

Comment:
- [ ] As per project requirements `All changes to the task list must be saved to local storage` However, I need to reload the page for the changes to occur, please kindly resolve the issue so that the Your book data is saved in the browser's memory without reloading the browser. 👍🏻 Here is a screenshot, your UI shows two books but your local store only has one book saved until I reload the page.


![image](https://user-images.githubusercontent.com/79058364/236280818-880b0c49-a3a8-42d8-a8e5-3557e93573ac.png)